src/drivers/esp32/src/esp_slip.c

```diff
@@ -33,6 +33,26 @@
 
 #define ESP_OVERHEAD_LEN 8
 
+typedef enum
+{
+  receiveStart,
+  receiveDirection,
+  receiveCommand,
+  receiveSize,
+  receiveValue,
+  receiveData,
+  receiveEnd,
+  error,
+} ESPblReceiveState;
+static ESPblReceiveState espblReceiveState = receiveStart;
+
+static uint8_t slipSize[2];
+static uint8_t slipValue[4];
+static uint8_t slipDataIndex = 0;
+static uint8_t slipSizeIndex = 0;
+static uint8_t slipValueIndex = 0;
+static bool inEscapeSequence = false;
```

Comment:
nit: this is not needed, all static variables are 0 by default.

Reply:
I like nits, makes me a better programmer!